Jenkinsfile_nightly

```diff
@@ -97,18 +97,18 @@ withNightlyPipeline(type, product, component) {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'build/reports/tests/integration/**/*'
   }
 
-  afterSuccess('build') {
+  afterAlways('build') {
```

Comment:
I think this change should be removed: as we need build t oscomplete for Integration tests to also be able to run.